yarn-project/end-to-end/src/integration_l1_publisher.test.ts

```diff
@@ -100,7 +145,7 @@ describe.skip('L1Publisher integration', () => {
   };
 
   it('Build 2 blocks of 4 txs building on each other', async () => {
-    const stateInRollup_ = await rollup.methods.rollupStateHash().call();
+    const stateInRollup_ = await rollup.read.rollupStateHash();
```

Comment:
Is `read` the viem way? 👀 I like it.

Reply:
Yes, viem is pretty nice. The only think which is a bit annoying is that sometimes the types get too complicated (see [this](https://github.com/AztecProtocol/aztec-packages/blob/4c821f7ba840dd2975ba19d4e5bbd8900378bb31/yarn-project/archiver/src/archiver/archiver.ts#L248) arg type).